app/src/main/AndroidManifest.xml

```diff
@@ -42,6 +42,17 @@
         <activity
             android:name=".twitterlogin.TwitterLoginRecipeActivity"
             android:screenOrientation="portrait" />
+        <activity
+            android:name=".instagramlogin.InstagramLoginRecipeActivity"
+            android:screenOrientation="portrait">
+
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="https" android:host="instagram.com" />
```

Comment:
O.O

Reply:
Because instagram api don't provide an interface or a button to do the login. Instead, its necessary to implements a WebView and catch the url with the access token